SecurityPkg/Library/Tcg2PreUefiEventLogLibNull/Tcg2PreUefiEventLogLibNull.c

```diff
@@ -0,0 +1,20 @@
+/** @file Tcg2PreUefiEventLogLibNull.c
+NOTE: NULL Tcg2PreUefiEventLogLibNull library instance
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Library/DebugLib.h>

```

Comment:
It looks like you can drop `DebugLib` to eliminate it as a dependency.

```suggestion
```

Reply:
Removed DebugLib from c and inf files